app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
See https://github.com/shakacode/bootstrap-sass-loader for more information on the bootstrap setup.